src/Chapter1.hs

```diff
@@ -641,7 +653,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | (abs n) < 10 = abs n
+  | otherwise = firstDigit $ flip div 10 $ abs n
```

Comment:
You can use infix notation here to write a bit cleaner code :slightly_smiling_face: 

```suggestion
  | otherwise = firstDigit $ abs n `div` 10
```